### PR TITLE
Finalize changelog for 7.5.0

### PR DESCRIPTION
**Pull request by mikemadden42**

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -155,28 +155,12 @@ https://github.com/elastic/beats/compare/v7.4.1...v7.5.0[View commits]
...
 
 - `kubernetes.container.id` field for `state_container` is deprecated in favour of ECS `container.id` and `container.runtime`. {pull}13884[13884]
 
-*Packetbeat*
-
-*Winlogbeat*
-
-*Functionbeat*
-
 ==== Known Issue
```
Comment (dedemorton) [inline CHANGELOG.asciidoc]: Delete this heading (the empty section is causing the doc build to fail).

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -14,8 +14,7 @@ https://github.com/elastic/beats/compare/v7.4.1...v7.5.0[View commits]
 - Libbeat HTTP's Server can listen to a unix socket using the `unix:///tmp/hello.sock` syntax. {pull}13655[13655]
 - Libbeat HTTP's Server can listen to a Windows named pipe using the `npipe:///hello` syntax. {pull}13655[13655]
 - Adding new `Enterprise` license type to the licenser. {issue}14246[14246]
-
-*Auditbeat*
+- By default, all Beats-created files and folders will have a umask of 0027 (on POSIX systems). {pull}14119[14119]
```
Comment (dedemorton) [inline CHANGELOG.asciidoc]: Is this true for all Beats? The PR made me think that individual Beats can override the default.

File: CHANGELOG.asciidoc [hunk 3]
```diff
@@ -14,8 +14,7 @@ https://github.com/elastic/beats/compare/v7.4.1...v7.5.0[View commits]
 - Libbeat HTTP's Server can listen to a unix socket using the `unix:///tmp/hello.sock` syntax. {pull}13655[13655]
```
Comment (dedemorton) [inline CHANGELOG.asciidoc]: If everything in this list is a breaking change, we need to do a better job of explaining how users are impacted and what they need to do to address the problem. It's not clear to me even after skimming the PR why these are considered breaking changes.

File: CHANGELOG.asciidoc [hunk 4]
```diff
@@ -60,10 +55,13 @@ https://github.com/elastic/beats/compare/v7.4.1...v7.5.0[View commits]
 - Fix azure fields names. {pull}14098[14098] {pull}14132[14132]
 - Fix calculation of `network.bytes` and `network.packets` for bi-directional netflow events. {pull}14111[14111]
 - Accept '-' as http.response.body.bytes in apache module. {pull}14137[14137]
+- Fix timezone parsing of MySQL module ingest pipelines. {pull}14130[14130]
+- Improve error message in s3 input when handleSQSMessage failed. {pull}14113[14113]
+- Fix race condition in S3 input plugin. {pull}14359[14359
```
Comment (dedemorton) [inline CHANGELOG.asciidoc]: missing close tag here

File: CHANGELOG.asciidoc [hunk 5]
```diff
@@ -72,12 +70,17 @@ https://github.com/elastic/beats/compare/v7.4.1...v7.5.0[View commits]
 - Change kubernetes.event.message to text {pull}13964[13964]
 - Fix performance counter values for windows/perfmon metricset.{issue}14036[14036] {pull}14039[14039] {pull}14108[14108]
 - Add FailOnRequired when applying schema and fix metric names in mongodb metrics metricset. {pull}14143[14143]
+- Convert indexed ms-since-epoch timestamp fields in `elasticsearch/ml_job` metricset to ints from float64s. {issue}14220[14220] {pull}14222[14222]
+- Fix ARN parsing function to work for ELB ARNs. {pull}14316[14316]
+- Update azure configuration example. {issue}14224[14224]
+- Limit some of the error messages to the logs only {issue}14317[14317] {pull}14327[14327]
+- Fix cloudwatch metricset with names and dimensions in config. {issue}14376[14376] {pull}14391[14391]
+- Fix marshaling of ms-since-epoch values in `elasticsearch/cluster_stats` metricset. {pull}14378[14378
```
Comment (dedemorton) [inline CHANGELOG.asciidoc]: missing close tag here too

File: CHANGELOG.asciidoc [hunk 6]
```diff
@@ -93,6 +96,7 @@ https://github.com/elastic/beats/compare/v7.4.1...v7.5.0[View commits]
 - Marking Central Management deprecated. {pull}14018[14018]
 - Add `keep_null` setting to allow Beats to publish null values in events. {issue}5522[5522] {pull}13928[13928]
 - Add shared_credential_file option in aws related config for specifying credential file directory. {issue}14157[14157] {pull}14178[14178]
+- Ensure that init containers are no longer tailed after they stop {pull}14394[14394]
```
Comment (dedemorton) [inline CHANGELOG.asciidoc]:
```suggestion
- Ensure that init containers are no longer tailed after they stop. {pull}14394[14394]
```

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (mikemadden42) [reply CHANGELOG.asciidoc]: I removed this section.  Good catch.

File: CHANGELOG.asciidoc (hunk 4 above)
Comment (mikemadden42) [reply CHANGELOG.asciidoc]: I added the missing tag.

File: CHANGELOG.asciidoc (hunk 5 above)
Comment (mikemadden42) [reply CHANGELOG.asciidoc]: I added the missing tag.

File: CHANGELOG.asciidoc (hunk 6 above)
Comment (mikemadden42) [reply CHANGELOG.asciidoc]: I added the period.

**Comment by dedemorton** [comment]
removing the needs_edit label because realistically I'm not going to get back to this.

**Pull request merged**